frontend/src/components/IngredientsList.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components/macro";
+import IngredientItem from "./IngredientItem";
+
+export default function IngredientsList({ ingredientsList }) {
+  console.log(ingredientsList);
```

Comment:
```suggestion

```

Reply:
Didn't saw that line during the clean up. Thanks